src/constants/Locales.js

```diff
@@ -0,0 +1,103 @@
+export const LOCAL_NUMERAL_FORMATS = {
```

Comment:
@wiadev  I have following questions regarding this file.
What's the reason for having those abbreviations there; i mean, where are they used?

IMHO, i think only the thousands separator and decimal separator are needed.

More, i think it would be better if those "locale" info would be provided in http://w101.metasfresh.com:8081/rest/api/userSession response.
wdyt?

Reply:
or we can even deliver the delimiters in layout element, i.e.

![image](https://user-images.githubusercontent.com/1244701/30276974-273a8944-970f-11e7-8351-9d5bb774d534.png)

Reply:
update: agreed with Mark to create a followup issue

Reply:
This is requirements of external library we're using for formatting - numeral.js.

Please check it there:  
http://numeraljs.com/#locales

@teosarca

Reply:
I saw this. But it contains precision only. We don't use abbreviations, they're placed only according to requirements, but not used in fact.